Solutions/GitHub/Hunting Queries/User Grant Access and Grants Other Access.yaml

```diff
@@ -0,0 +1,26 @@
+id: f18c4dfb-4fa6-4a9d-9bd3-f7569d1d685a
+name: GitHub User Grants Access and Other User Grants Access
+description: |
+  'This hunting query identifies Accounts in GitHub that have granted access to another account which then grants access to yet another account that may be a sign of compromise.'
+requiredDataConnectors: []
```

Comment:
Please add the connectorid and datatype in this format
requiredDataConnectors: -
connectorId: connector id
dataTypes: - name of the parser which we use in the query

Thanks,
Aditi Maheshwari

Reply:
Added